SDPi_Supplement/asciidoc/volume1/tf1-ch-a-requirements-interoperability.adoc

```diff
@@ -12,6 +12,147 @@ The content for this appendix may be pushed to SDPi 1.x.
+
+Note:  The implementation of this framework will be extended as the specifications and tooling mature.
+
 // A.2
 === Integrating Safety, Effectiveness & Security Requirements & Considerations
 
+In 2007, a joint effort between ISO/TC 215 and IEC/SC 62A was launched --- Joint Working Group 7 (JWG7) -- to focus on how to apply risk management to medical devices and health information systems and software that needed to interoperate on shared (hospital owned & managed) networking infrastructure.  The resulting standard, <<iec_80001_1_2021>>, with a first edition published in 2010 and revised in 2021, not only provided a process for performing coordinated multi-stakeholder risk management for these technologies, but recognized the three *key properties* that would be the focus of that risk management:  *_Safety, Effectiveness & Security (SES)_* footnote:ses_definitions[For definitions of these and other related terms, consult the https://81001.org[NHS 81001.org web page.] Last accessed 2022.10.04.].
+
+During the development of the 80001-1 standard, though, it was recognized that risk management is just one of a number of other core themes that had to be managed in concert (e.g., quality management, human factors / usability).  Also ensuring SES required processes that involve a total product lifecycle, with responsibilities transitioning across that period.  To address these requirements, JWG7 developed the <<iso_81001_1_2021>> standard, which also included The Temple diagram to communicate the various aspects of SES that must be considered and managed:
+
+.ISO/IEC 81001-1 _The Temple Diagram_
+
+image::../images/vol1-diagram-81001-temple.svg[algin=center]
+
+#TODO:  SHOULD THE FOLLOWING SOURCE LINE BE A FOOTNOTE ... HUNG ON WHERE ... IHE CONVENTION?#
+
+[none]
+. Source:  <<iso_81001_temple>>
+
+One of the key challenges for implementing this standard, though, is what might be labeld:  *_The Interoperability Trust Gap_*.  This is the technology "hand off" space between the left side of the lifecycle -- Design and development phase, where key responsibility is by each of the "Accountable Manufacturer" organizations, and the right side of the lifecycle -- Implementation phase & Clinical use phase, where key responsibility is on the Accountable Healthcare Delivery Organization (HDO).  Though this reads well in the standards and the model organizes everything in a clear fashion, operationalizing this in real world use remains a Sisyphean effort, primarily due to the amount of expertice, time and resources needed to effectively implement the SES standards as part of normal operating business in HDOs.
+
+To address this SES implementation problem, the SDPi profiles:
+
+. Leverage the ISO/IEEE 11073-1070x Participant Key Purposes (PKP) standards, which represent a consensus standard for risk management of technologies that are implemented in that left-right gap on the Temple model
```

Comment:
PKP, ICS, RI, SES, MDI and other abbrevs should be introduced in the glossary and linked in the text via `<<acronym_[...]>>`